docs/framework/react/guides/basic-concepts.md

```diff
@@ -92,17 +92,42 @@ const {
 } = field.state
 ```
 
-There are three field states that can be useful to see how the user interacts with a field: A field is _"touched"_ when the user clicks/tabs into it, _"pristine"_ until the user changes value in it, and _"dirty"_ after the value has been changed. You can check these states via the `isTouched`, `isPristine` and `isDirty` flags, as seen below.
+There are three states in the metadata that can be useful to see how the user interacts with a field:
+
+- _"touched"_, after the user clicks/tabs into the field
+- _"pristine"_, until the user changes the field value
+- _"dirty"_, after the fields value has been changed
+
+You can check these states via the `isTouched`, `isPristine`, and `isDirty` flags, as seen below.
```

Comment:
Why not directly using the flag names in the bullet point so we can avoid the other sentence?

Also we might mention `isDefaultValue` here